docs/UserGuide.adoc

```diff
@@ -96,24 +100,28 @@ Examples:
 
 === Check-in room: `checkin`
 
-Checks in the room's first active booking. A booking is considered **active** if
-its start and end date includes today's date, inclusively.
-The guest who made the booking will be added to both the checked-in guest list and the archived guest list. +
+Checks in the room's first active booking.
+A booking is considered **active** if its start and end date includes today's date, inclusively.
+Upon checking in the room's first active booking, the guest who made the booking will be added to
+the checked-in guest list. +
 Format: `checkin r/ROOM_NUMBER`
 
 Examples:
```

Comment:
Can include an example that demonstrates the possibility of checking in the same guest twice.